src/components/menu/ModelizerSettingsMenu.vue

```diff
@@ -73,6 +91,9 @@ const menuItems = computed(() => [
     visible: true,
   },
 ]);
+const userInitials = computed(() => (user.value?.firstname && user.value?.lastname
+  ? `${user.value.firstname.at(0)}${user.value.lastname.at(0)}`
+  : ''));
```

Comment:
: null at the end

Reply:
done

Reply:
You simplify the code, but I prefere you to keep previous version with this changes:
```
const userInitials = computed(() => (user.value?.firstname && user.value?.lastname
  ? `${user.value.firstname.at(0)}${user.value.lastname.at(0)}`
  : t('page.modelizer.settings.user.unknownInitials')));
```

Reply:
My bad, I don't see the v-if.

Reply:
nope